plog-kafka/src/main/java/com/airbnb/plog/kafka/KafkaHandler.java

```diff
@@ -46,7 +71,14 @@ private static JsonObject meterToJsonObject(Meter meter) {
     @Override
     protected void channelRead0(ChannelHandlerContext ctx, Message msg) throws Exception {
         seenMessages.incrementAndGet();
-        final byte[] payload = msg.asBytes();
+        byte[] payload = msg.asBytes();
+        if (encryptionConfig != null) {
+            try {
+                payload = encrypt(payload);
+            } catch (Exception e) {
+                log.error("Fail to encrypt message: ", e);
```

Comment:
this might not print useful information, i guess it will print something like, "Fail to entry message: XXException@fjdsl"